pkg/installer/install.go

```diff
@@ -22,7 +24,7 @@ func (in *Installer) Install(upgrade bool) error {
 
 			errChan <- in.installEtcd()
 		}()
-	} else if !in.stosConfig.Spec.IncludeEtcd && !upgrade {
+	} else if !in.stosConfig.Spec.IncludeEtcd && !in.stosConfig.Spec.Install.DryRun && !upgrade {
```

Comment:
nit: the check for `!in.stosConfig.Spec.IncludeEtcd` isn't necessary, as the earlier if already checks that its true.

Removing it might make it easier to determine what we're checking for

Reply:
Good spot 👍